src/current/releases/cloud.md

```diff
@@ -14,6 +14,12 @@ Get future release notes emailed to you:
 
 {% include marketo.html formId=1083 %}
 
+## July 23, 2024
+
+<h3 id="2024-07-23-general-updates"> General updates </h3>
+
+- When creating a new CockroachDB {{ site.data.products.dedicated }} cluster, you can now [select the cluster's major version]({% link cockroachcloud/create-your-cluster.md %}#step-8-select-the-cockroachdb-version), which defaults to the latest stable version. At any given time, you may be able to choose from additional stable releases or experimental [Pre-Production Preview]({% link cockroachcloud/upgrade-policy.md %}#pre-production-preview-upgrades) releases. Patch releases within a major version, including subsequent patches after a Pre-Production Preview, are applied automatically.
```

Comment:
```suggestion
- When creating a new CockroachDB {{ site.data.products.dedicated }} cluster, you can now [select the cluster's major version]({% link cockroachcloud/create-your-cluster.md %}#step-8-select-the-cockroachdb-version). The default option is the latest stable version. At any given time, you may be able to choose from additional stable releases or experimental [Pre-Production Preview]({% link cockroachcloud/upgrade-policy.md %}#pre-production-preview-upgrades) releases. Patch releases within a major version are applied automatically.
```

Reply:
I don't think we need to get into the further detail on PPP. Already documented and linked. Don't want to distract; it's not applicable to most customers.